src/ng/cookieReader.js

```diff
@@ -22,6 +22,13 @@ function $$CookieReader($document) {
     }
   }
 
+  //null object implementation in case document was mocked out
+  if (!rawDocument || typeof rawDocument.cookie !== 'string') {
+    return function() {
+      return lastCookies;
+    };
+  }
+
```

Comment:
Ugh! Mocking window without a document is surely crazy juice!
OK let's go with this version.